### PR TITLE
:sparkles: [#1046] Zaken list extra ordering options

### LINKED ISSUE #1046: As a developer, I want to by able to order Zaken list by more fields
... so I don't have to implement this ordering in an API client

This is a part of adding support for Zaken API 1.3
Main issue - https://github.com/open-zaak/open-zaak/issues/1407

**Description**
See upstream issue: https://github.com/VNG-Realisatie/gemma-zaken/issues/1892

### PR DESCRIPTION
**Pull request by stevenbal**
Fixes #1046

PR in reference implementation has been merged: https://github.com/VNG-Realisatie/zaken-api/pull/206

**Changes**
* Add extra options for Zaken list ordering: einddatum, publicatiedatum, archiefactiedatum
* Explicitly document these options in the Zaken OAS

### CODE REVIEW

**Comment by stevenbal** [comment]
replacement for https://github.com/open-zaak/open-zaak/pull/1045

File: src/openzaak/components/zaken/api/filters.py [hunk 1]
```diff
@@ -44,6 +46,12 @@ class ZaakFilter(FilterSet):
         field_name="rol__organisatorischeeenheid__identificatie",
         help_text=get_help_text("zaken.OrganisatorischeEenheid", "identificatie"),
     )
+    ordering = filters.OrderingFilter(
+        fields=("startdatum", "einddatum", "publicatiedatum", "archiefactiedatum",),
+        help_text=_(
+            mark_oas_difference("Het veld waarop de resultaten geordend worden.")
```
Comment (annashamray) [inline src/openzaak/components/zaken/api/filters.py:52]: I think it's not a deviation from OAS anymore, so this mark can be removed

**Review by annashamray — CHANGES_REQUESTED** [review]
Could you please rebase on `main` one more time? Just to make sure that the CI will pass

**Comment by annashamray** [comment]
replace with new PR #1445

**Pull request closed without merging**